l10n.js

```diff
@@ -219,40 +223,70 @@ document.webL10n = (function(window, document, undefined) {
               parsedRawLinesCallback();
               return;
             }
-            var line = entries.shift();
```

Comment:
Instead of removing this whole block and introducing more helper functions, consider using something like this instead:

```js
if (entries.length && line.charAt(-1) === '\\') {
  entries[0] = line.slice(0, -1) + entries[0];
  continue;
}
```

The previous snippet just illustrates my thought. Don't forget to address edge cases such as:
- Dealing with escaped backslashes (i.e. `This is not escaping an EOL --> \\`).
- Comments with a `\` at the end, are they also escaping an EOL?
- Comments after a EOL. Are they part of the previous line or should they be ignored.
- ...